src/services/twilio.ts

```diff
@@ -0,0 +1,53 @@
+import { getTwilioConfig } from "@config/app"
+import { UnknownPhoneProviderServiceError } from "@domain/phone-provider"
+import { baseLogger } from "@services/logger"
+import twilio from "twilio"
+
+export const TwilioClient = (): IPhoneProviderService => {
+  const client = twilio(getTwilioConfig().apiKey, getTwilioConfig().apiSecret, {
+    accountSid: getTwilioConfig().accountSid,
+  })
+
+  const sendText = async ({ body, to, logger }) => {
```

Comment:
same comment about signature, we should try to keep the same signature

Reply:
is this the fix you're looking for? https://github.com/GaloyMoney/galoy/pull/730/commits/bfce0ba30bcf717d31ef4744c9fe858b16d1a493

Reply:
please check this comment https://github.com/GaloyMoney/galoy/pull/730#discussion_r757807499